tests/console/docker.pm

```diff
@@ -62,8 +62,8 @@ sub run {
     #   - pull image of last released version of openSUSE Leap
     my $last_released_leap_version = '42.3';
     assert_script_run("docker image pull opensuse:$last_released_leap_version");
-    #   - pull image of openSUSE Tumblewedd
-    assert_script_run('docker image pull opensuse:tumbleweed');
+    #   - pull image of openSUSE Tumbleweed
+    assert_script_run('docker image pull opensuse:tumbleweed', timeout => 180);
```

Comment:
What about `assert_script_run("docker image pull opensuse:$last_released_leap_version");` ?

Reply:
The video doesn't show any timeout there

Reply:
Yes, but that's probably just luck:
```
~ λ docker pull opensuse:tumbleweed
tumbleweed: Pulling from library/opensuse
bfb02aa233ba: Extracting [========================>          ] 20.45MB/41.91MB

~ λ docker pull opensuse:42.3
42.3: Pulling from library/opensuse
b28a1d8bf9e0: Downloading [====>                             ] 3.661MB/44.65MB
```

But if you prefer I can merge and maybe ping you later ;)

Reply:
Dockerhub speed is not very reliable, we had timeouts pulling nginx:alpine - which is quite small